app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(cp: str) -> None:
+    command, file_in, file_out = cp.split()
```

Comment:
Ensure that the input string contains exactly three elements separated by spaces, and that the first element is 'cp'. This is a requirement from the checklist to ensure the command is valid.